src/components/ImageDisplay/ThumbnailRow.vue

```diff
@@ -1,22 +1,37 @@
 <template>
-  <div class="images">
-    <img
-      v-for="(item, index) in images"
-      :key="index"
-      :src="thumbnailWithFallback(item)"
-      onerror="this.onerror=null;this.src='https://via.placeholder.com/60/FF0000/FFFFFF?text=jpg'"
-      :title="item.base_filename"
-      :class="{'selected_thumbnail' : item.image_id == selected_image}"
-      loading="lazy"
-      class="recent-image"
-      @click="setActiveImage(item)"
+  <div
+    v-if="Object.keys(grouped_images).length > 0"
+    class="images"
+  >
+    <div
+      v-for="(item, SMARTSTK) in grouped_images"
+      :key="SMARTSTK"
+      class="image-container"
     >
+      <img
+        v-if="SMARTSTK !== 'site'"
```

Comment:
So this is partially my bad, but having to check our SMARTSTK value feels like a code smell - I think it's going to confuse future-me. As it stands, we have a JSON object of:

```
{'site': mrc1,
 1234: [...grouped images...],
 5678: [...grouped images...]
}
```

etc...

It might be cleaner to have a structure like 

```
{'site': mrc1,
 'imageGroups': [ 
    1234: [...grouped images...],
    5678: [...grouped images...],
    ...]
}
```

so that your v-for loop becomes `v-for="item in grouped_images.imageGroups"` and then you don't have to check if the SMARTSTK key is site! I don't think you'll even need the SMARTSTK key anymore

Reply:
Great idea. Seems a lot more organized. Done!